packages/mobile/src/app/App.test.tsx

```diff
@@ -6,7 +6,7 @@ import 'react-native'
 // Note: test renderer must be required after react-native.
 import renderer from 'react-test-renderer'
 
-import { App } from 'app/app'
+import { App } from './App'
```

Comment:
wow it actually renders!

Reply:
Is our goal to render the whole app and test it in an integration or e2e style fashion?
In my experience, the smaller the scope for your tests, the less you need to do things like mocking all of the external libraries and browser API usage etc.

Reply:
I think some high level integration / smoke tests would be good to have, seeing as we have no tests for mobile at all right now. We at least have probers for web. But generally I agree that the scope of most tests should be small, the whole reason I went down this path was to unit test the deep-linking code lol

Reply:
Yeah agreed we can start running smaller tests now, but it's great you covered all the bad cases that we might experience along the way.